helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -97,7 +97,9 @@ internal object PathExcludeGenerator {
             files.filter { pathExcludeExclude.matches(it.path) }.toSet()
         }
 
-        return greedySetCover(filesForPathExcludes).toSet()
+        return greedySetCover(filesForPathExcludes) { a, b ->
+            b.pattern.length - a.pattern.length
```

Comment:
Just a thought: I wonder whether "shorter" is the best metric to identify "simpler" patterns (if getting "simpler" patters from a human perspective is the goal). Instead, maybe "contains fewer wildcards" could be an alternative metric.